src/lib/layouts/DocsArticle.svelte

```diff
@@ -117,4 +117,13 @@
             </div>
         </aside>
     </article>
-</main>
\ No newline at end of file
+</main>
+
+<style>
+    @media (min-width: 1280px) and (max-width: 1330px) {
+        .web-article-content {
+            /* original/default is 41.5rem */
+            max-inline-size: 40.5rem !important;
```

Comment:
This seems like it causes other content to be misaligned. For example, [Auth docs](http://1228.gkswsco.37.27.92.31.sslip.io/docs/products/auth):
<img width="664" alt="image" src="https://github.com/user-attachments/assets/01da1c4c-2494-4cd6-97c5-e26514659dbc">

Reply:
Done.